ixmp/tests/test_tutorials.py

```diff
@@ -7,53 +7,73 @@
 
 from ixmp.testing import get_cell_output, run_notebook
 
+group_base_name = platform.system() + platform.python_version()
+
+GHA = "GITHUB_ACTIONS" in os.environ
```

Comment:
Typically we'd define this in `ixmp.testing`. If it's only used here for now, fine to leave it; but if it's to be re-used, we can move it there.